index.jsx

```diff
@@ -137,11 +137,19 @@ module.exports = function Dimensions ({
         this.getWindow().removeEventListener('resize', this.onResize)
       }
 
+      getWrappedInstance = () => this.refs.wrappedInstance;
```

Comment:
Cool. Updated in master. If you have a chance to write a JSDoc comment above the function I can generate a new README and push a new version, if not I'll get round to it in the next week or two.